src/pages/Meeting/DetailMeatingManagePage.tsx

```
@@ -158,25 +194,25 @@ export default function DetailMeetingManagePage() {
               토론 참여자
             </h2>
-
-            <div className="bg-white border-[2px] border-[#EAE5E2] rounded-[12px] p-6 min-h-[540px]" >
+            <div className="min-h-[600px] bg-white border-[2px] border-[#EAE5E2] rounded-[12px] p-6 min-h-[540px]" >
               <div className="space-y-4">
-                {participants.map((member: ClubMember) => (
+                {participants.map((member: MeetingMemberItem) => (
                   <div className="flex items-center justify-between border-b border-[#EAE5E2] pb-3">
                     <div className="flex items-center gap-4 min-w-[180px]">
-                      <img src={member.basicInfo.profileImageUrl} className="w-[36px] h-[36px] rounded-full bg-[#DADADA]" />
+                      <img src={member.memberInfo.profileImageUrl || "/assets/basic_profile.png"}
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)